desktop/packages/linux/rs.ruffle.Ruffle.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Name=Ruffle
+GenericName=Flash Player
+Comment=Flash Player emulator
+Icon=rs.ruffle.Ruffle
+Exec=ruffle %u
+MimeType=application/x-shockwave-flash;application/vnd.adobe.flash.movie
+Categories=Game;Utility;Graphics;AudioVideo;Player;Viewer
+Keywords=flash;swf
```

Comment:
Maybe `emulator` too? Dunno if it's used for any other application though.

Reply:
Done!